src/com/oltpbenchmark/api/Loader.java

```diff
@@ -160,7 +160,7 @@ public void load(Connection conn) {
     }
 
     private PreparedStatement getInsertStatement(Connection conn, String tableName) throws SQLException {
-      return conn.prepareStatement(SchemaManager.getInsertDml(tableName));
+      return conn.prepareStatement(DefaultSchemaManager.getInsertDml(tableName));
```

Comment:
should this be always using DefaultSchemaManager? this is a bit confusing. if we want to use the same getInsertDml logic in both cases, we should at least park the static method in some class that is not semantically specific about the use case (i.e. DefaultSchemaManager semantically specifies it should be used in the non-geo-partitioned case, so the reader could think this is a mistake)

Reply:
Nice catch. I moved it to TPCCTableSchema instead